apps/vscode/extension/src/WebViewMessageHandler.ts

```diff
@@ -194,3 +219,22 @@ export class WebViewMessageHandler {
 		}
 	}
 }
+
+function getMimeTypeFromPath(filePath: string): string {
+	const extension = filePath.split('.').pop()?.toLowerCase()
+	if (!extension) throw new Error('No extension found in file path')
+
+	const mimeTypes: Record<string, string> = {
+		jpg: 'image/jpeg',
+		jpeg: 'image/jpeg',
+		png: 'image/png',
+		webp: 'image/webp',
+		gif: 'image/gif',
+		apng: 'image/apng',
+		avif: 'image/avif',
+		svg: 'image/svg+xml',
+	}
+	const mimeType = mimeTypes[extension]
```

Comment:
might be also worth checking `DEFAULT_SUPPORTED_IMAGE_TYPES` (or `DEFAULT_SUPPORTED_MEDIA_TYPE_LIST` if we support videos too, and maybe support audio one day)

Reply:
Added some logic for this now 👍